### PR TITLE
remove verbose mode for rsync

### PR DESCRIPTION
**Pull request by pfefferle**
I had problems with travis and the verbose mode of `rsync`:

> The job exceeded the maximum log length, and has been terminated.

I would suggest to remove the flag, or at least to hide it behind an `env` var.

### CODE REVIEW

**Comment by getsource** [comment]
I like the idea of having an environment variable to toggle it. Maybe something referencing verbose, or perhaps enabling "debug"?

**Comment by pfefferle** [comment]
How should I call the var?

* `$WPT_VERBOSE`
* `$WPT_VERBOSE_LOGGING`
* `$WPT_VERBOSE_OUTPUT`
* ...

any suggestions?

**Comment by pfefferle** [comment]
I made some changes... It is easier, to discuss about code ;)

**Comment by getsource** [comment]
Thanks much! What do you think about `WPT_DEBUG`?

I'll leave a couple comments in the PR as well on specifics.

File: prepare.php [hunk 1]
```diff
@@ -15,6 +15,7 @@
 $WPT_SSH_OPTIONS = getenv( 'WPT_SSH_OPTIONS' ) ? : '-o StrictHostKeyChecking=no';
 $WPT_TEST_DIR = getenv( 'WPT_TEST_DIR' );
 $WPT_PHP_EXECUTABLE = getenv( 'WPT_PHP_EXECUTABLE') ? : 'php';
+$WPT_LOGGING = getenv( 'WPT_LOGGING') ? : 'normal';
```
Comment (getsource) [inline prepare.php:18]: If we're not using the value "normal" for anything, would it make sense to leave it empty for the default value?

File: report.php [hunk 2]
```diff
@@ -14,6 +14,7 @@
 $WPT_PREPARE_DIR = getenv( 'WPT_PREPARE_DIR' );
 $WPT_SSH_OPTIONS = getenv( 'WPT_SSH_OPTIONS' );
 $WPT_REPORT_API_KEY = getenv( 'WPT_REPORT_API_KEY' );
+$WPT_LOGGING = getenv( 'WPT_LOGGING') ? : 'normal';
```
Comment (getsource) [inline report.php:17]: Nit-pick, but this could use a space after `'WPT_LOGGING'` for coding standards reasons.

Comment (pfefferle) [reply report.php:17]: oops

File: prepare.php [hunk 3]
```diff
@@ -15,6 +15,7 @@
 $WPT_SSH_OPTIONS = getenv( 'WPT_SSH_OPTIONS' ) ? : '-o StrictHostKeyChecking=no';
 $WPT_TEST_DIR = getenv( 'WPT_TEST_DIR' );
 $WPT_PHP_EXECUTABLE = getenv( 'WPT_PHP_EXECUTABLE') ? : 'php';
+$WPT_DEBUG = getenv( 'WPT_DEBUG') ? : '';
```
Comment (getsource) [inline prepare.php:18]: Would there be an issue with having this get set to false if it isn't assigned, rather than setting an empty string?

Comment (getsource) [reply prepare.php:18]: Ah, nit-pick, but looks like this may also need a space after `'WPT_DEBUG'` before the parenthesis.

File: report.php [hunk 4]
```diff
@@ -14,6 +14,7 @@
 $WPT_PREPARE_DIR = getenv( 'WPT_PREPARE_DIR' );
 $WPT_SSH_OPTIONS = getenv( 'WPT_SSH_OPTIONS' );
 $WPT_REPORT_API_KEY = getenv( 'WPT_REPORT_API_KEY' );
+$WPT_DEBUG = getenv( 'WPT_DEBUG' ) ? : '';
```
Comment (getsource) [inline report.php:17]: Same question with the default as above.

**Comment by getsource** [comment]
Yes, it's working for me. I may be missing something, so apologies if so!
I think this is a bit more consistent with the way the rest of the string variables are handled in code so far.

If it doesn't exist at all (not in `.env` at all), it will end up `false` (bool).
if it is defined in `.env` as empty, it ends up as an empty string, 

In both cases it's "set" and "empty".

**Comment by getsource** [comment]
I'm seeing the 👍 but didn't want to assume incorrectly -- if you'd like to merge, go ahead -- if not, either I can, or of course we can keep discussing.

It looks good to me -- thank you!

**Comment by pfefferle** [comment]
I like your change and I am fine with merging it 😊

**Comment by pfefferle** [comment]
I am the MR owner, so I cannot approve the changes.

**Pull request merged**